src/assets/CollectiblesController.ts

```diff
@@ -118,6 +118,12 @@ export interface CollectibleMetadata {
   collectionImage?: string;
 }
 
+interface DetectionParams {
+  autodetected: boolean;
+  address: string;
+  chainId: string;
+}
+
```

Comment:
nit: Should this `interface` be at the `CollectibleDetectionController` file?

Reply:
Good question, I'd say since it is used in this controller it should be here.
For example someone wants to use a different method to autodetect: let's say a button which performs a simple API call and with the response data they call `addCollectible`, they need to use that interface.